sys-libs/libnvidia-container/files/libnvidia-container-1.17.0-fix-makefile-r1.patch

```diff
@@ -0,0 +1,220 @@
+Base on The patch from Arch[1] but better suited to Gentoo needs.
+
+[1]: https://gitlab.archlinux.org/archlinux/packaging/packages/libnvidia-container/-/blob/main/fix-makefile.patch
```

Comment:
Good, I am glad you linked it. Sometimes people forget to do that. Consider changing it to be a commit (so s/main/SOME_COMMIT_YOU_CHECKED/) so that the link works forever.